### ISSUE TITLE
Make is not idempotent

### ISSUE THREAD

**Issue by tiveront** [red-4/curious-moon#54]
Issue : import.master_plan is not dropped on import.

In the makefile
`import: master
@echo "COPY import.master_plan FROM $(CSV) WITH DELIMITER ',' HEADER CSV;" >> $(BUILD)`

In 02-normalization/scripts/import.sql
`drop table if exists master_plan;
create table master_plan(...);`

If you run `make clean && make` multiple times the import.master_plan continues to grow each time as it isn't dropped.

Suggested fix is to update the import.sql:
`drop table if exists import.master_plan;
create table import.master_plan(...);`

Might also add the schema creation to the import.sql too
`create schema if not exists import;`

**Comment by robconery** [comment]
Yeah I have to accommodate both scenarios - with a schema and without. What I was hoping to get across with the comment at the bottom of 02_build.sql is that you would need to add the schema to the name of the table. You can change this on your own with your scripts to make it work. I didn't want to do it straight away (if memory serves) as the chapter builds from no schema to working with one.
